website/sales/api/v2/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import path
+
+from sales.api.v2.views import (
+    ShiftListView,
+    ShiftDetailView,
+    OrderListView,
+    OrderDetailView,
+)
+
+urlpatterns = [
+    path("sales/", ShiftListView.as_view(), name="shift-list"),
+    path("sales/<int:pk>/", ShiftDetailView.as_view(), name="shift-detail",),
+    path("sales/<int:pk>/orders/", OrderListView.as_view(), name="shift-orders",),
```

Comment:
```suggestion
    path("sales/shifts/<int:pk>/", ShiftDetailView.as_view(), name="shift-detail",),
    path("sales/shifts/<int:pk>/orders/", OrderListView.as_view(), name="shift-orders",),
```

The object needs to be in the URL. Otherwise you are not getting anything.